Library/Formula/fig.rb

```diff
@@ -56,12 +54,9 @@ def install
       r.stage { system "python", "setup.py", "install", "--prefix=#{libexec}" }
     end
 
-    system "python", "setup.py", "install", "--prefix=#{prefix}"
-
-    rm "#{lib}/python2.7/site-packages/site.py"
-    rm "#{lib}/python2.7/site-packages/easy-install.pth"
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
 
-    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    (bin/"fig").write_env_script libexec/"bin/fig", :PYTHONPATH => ENV["PYTHONPATH"]
```

Comment:
Dependent packages should be installed to `libexec/"vendor"` to prevent these conflicts; there's advice on that in https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md. Thanks for catching the conflict.